client/v3/config_test.go

```diff
@@ -12,83 +12,92 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package command
+package clientv3
 
 import (
 	"crypto/tls"
-
-	"go.uber.org/zap"
 	"testing"
 	"time"
 
 	"github.com/stretchr/testify/assert"
 	"go.etcd.io/etcd/client/pkg/v3/transport"
-	clientv3 "go.etcd.io/etcd/client/v3"
+	"go.uber.org/zap"
 )
 
 func TestNewClientConfig(t *testing.T) {
 	cases := []struct {
 		name         string
-		spec         clientv3.ConfigSpec
-		expectedConf clientv3.Config
+		spec         ConfigSpec
+		expectedConf Config
 	}{
 		{
-			name: "default secure transport",
-			spec: clientv3.ConfigSpec{
+			name: "only has basic info",
+			spec: ConfigSpec{
 				Endpoints:        []string{"http://192.168.0.10:2379"},
 				DialTimeout:      2 * time.Second,
 				KeepAliveTime:    3 * time.Second,
 				KeepAliveTimeout: 5 * time.Second,
-				Secure: &clientv3.SecureConfig{
-					InsecureTransport: false,
-				},
 			},
-			expectedConf: clientv3.Config{
+			expectedConf: Config{
 				Endpoints:            []string{"http://192.168.0.10:2379"},
 				DialTimeout:          2 * time.Second,
 				DialKeepAliveTime:    3 * time.Second,
 				DialKeepAliveTimeout: 5 * time.Second,
-				TLS:                  &tls.Config{},
 			},
 		},
 		{
-			name: "default secure transport and auth enabled",
-			spec: clientv3.ConfigSpec{
+			name: "auth enabled",
```

Comment:
Note, Watch out for reordering test. I will review it, but it makes it inconvinient.

Reply:
The sub tests are not reordered.  I just added two more sub tests and renamed one of them.